Mathlib/RingTheory/Kaehler/JacobiZariski.lean

```diff
@@ -0,0 +1,567 @@
+/-
+Copyright (c) 2024 Andrew Yang. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Andrew Yang
+-/
+import Mathlib.RingTheory.Kaehler.CotangentComplex
+import Mathlib.RingTheory.Generators
+import Mathlib.Algebra.Module.SnakeLemma
+
+/-!
+
+# The Jacobi-Zariski exact sequence
+
+Given `R → S → T`, the Jacobi-Zariski exact sequence is
+```
+H¹(L_{T/R}) → H¹(L_{T/S}) → T ⊗[S] Ω[S/R] → Ω[T/R] → Ω[T/S] → 0
+```
+The maps are
+- `Algebra.H1Cotangnet.map`
+- `Algebra.H1Cotangnet.δ`
+- `Algebra.H1Cotangnet.δ`
```

Comment:
```suggestion
- `Algebra.H1Cotangent.map`
- `Algebra.H1Cotangent.δ`
- `Algebra.H1Cotangent.δ`
```

Reply:
I think these 5 maps should be `map`, `δ`, `mapBaseChange`, `map`, `map` respectively? And maybe the lemma `exact_δ_map` should be named `exact_δ_mapBaseChange`?